.github/workflows/cmake.yml

```diff
@@ -13,27 +13,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - ubuntu-18.04
-          - macOS-10.15
-        cmake_opts: [ "" ]
-        arch: [ x86_64 ]
-        toolchain: [ default.cmake ]
-
         include:
+          - os: macOS-10.15
```

Comment:
I like this, but add a name to each variant so it is easier to identify in CI `${{ matrix.name }}`

Reply:
Thanks. I moved the workflow changes into a separate PR.